src/quo/components/wallet/address_text/view.cljs

```diff
@@ -8,26 +9,37 @@
             [utils.address :as utils]))
 
 (defn- colored-network-text
-  [theme network]
+  [{:keys [theme network size weight]}]
   (let [{:keys [network-name short-name]} network]
     [text/text
-     {:size  :paragraph-2
-      :style {:color (colors/resolve-color network-name theme)}}
-     (str short-name ":")]))
+     {:size   size
+      :weight weight
+      :style  {:color (colors/resolve-color (or network-name (keyword network)) theme)}}
+     (str (or short-name network) ":")]))
 
 (defn- view-internal
-  [{:keys [networks address blur? theme format]}]
-  (let [network-text-xf (map #(colored-network-text theme %))
-        address-text    [text/text
-                         {:size   :paragraph-2
-                          ;; TODO: monospace font
-                          ;; https://github.com/status-im/status-mobile/issues/17009
-                          :weight :monospace
-                          :style  (style/address-text format blur? theme)}
-                         (if (= format :short)
-                           (utils/get-short-wallet-address address)
-                           address)]]
-    (as-> networks $
+  [{:keys [networks address blur? theme format full-address? size weight]
+    :or   {size :paragraph-2}}]
+  (let [network-text-xf                      (map #(colored-network-text {:theme   theme
```

Comment:
that's quite a lot of work, maybe worth considering doing that in a dedicated sub, but don't have much context

Reply:
what is `-xf`?

Reply:
I renamed to `network-colored-text`